src/lms-ds-loader/lms_ds_loader/arguments.py

```diff
@@ -18,45 +22,17 @@ class Arguments:
         Database engine, either "mssql" or "postgresql"
     """
 
-    def __init__(self, csv_path, engine):
-        assert csv_path is not None, "csv_path cannot be None"
-        assert csv_path.strip() != "", "csv_path cannot be whitespace"
-        assert engine is not None, "engine cannot be None"
-        assert engine.strip() != "", "engine cannot be whitespace"
-
-        self.csv_path = csv_path
-        self.engine = engine
-        self.connection_string = ""
-
-    @staticmethod
-    def _build_for_mssql_integrated_security(server, port, db_name):
-        assert server is not None
-        assert server.strip() != ""
-
-        assert db_name is not None
-        assert db_name.strip() != ""
-
-        if not port:
-            port = 1433
-        if type(port) == str and port.strip() == "":
-            port = 1433
-
-        return f"mssql+pyodbc://{server},{port}/{db_name}?driver=SQL Server?Trusted_Connection=yes"
+    csv_path: str
+    engine: str
 
     @staticmethod
-    def _build_for_mssql(server, port, db_name, username, password):
-        assert server is not None
-        assert server.strip() != ""
-
-        assert db_name is not None
-        assert db_name.strip() != ""
-
+    def _get_mssql_port(port):

```

Comment:
Reading that this static method starts with _ I am assuming it should be internal or not intended to be used outside of the function. I'd put this function outside of the class... So when someone imports the Arguments class, won't be able to do Arguments._get_mssql_port() from outside the class.

Reply:
Not intended for use outside of this class. But it is not critical to _prevent_ it from being used.  I can see that it is possible to move it out of the class. I tried that and all tests pass. 

Where does it belong though? If it is truly related to "arguments", even if static, then I think it belongs with the class. But otherwise, it could just be a bare function in the module. In this case, it is hard to tell - just not very clear cut. 

I'm inclined to leave it alone unless anyone else has a strong preference to move it out of the class.

Reply:
I probably would have put at the module level, but no strong preference.